miniapps/lammps/lammpsDriver.cxx

```diff
@@ -126,6 +126,9 @@ int main(int argc, char **argv)
   MPI_Comm sim_comm = MPI_COMM_WORLD;
   MPI_Comm_rank(sim_comm, &globalInfo.me);
 
+  // Initialize Profiler
```

Comment:
@inealey I am not sure the sensei namespace is visible here. Did you try building after the change?

Reply:
Victor poked a bit at other examples and we think it may work here. Accepting the PR to test it might be the quicker way to know.